src/infra/redis/redis.service.ts

```diff
@@ -7,13 +6,12 @@ import { RedisAdapter } from './interfaces/index.js';
 import { IoRedisAdapter } from './ioredis.adapter.js';
 import { RedisConfig } from './redis.config.js';
 
-@Injectable()
-export class RedisService {
+export class RedisFactory {
 	public constructor(
 		private readonly config: RedisConfig,
 		private readonly logger: Logger,
 	) {
-		this.logger.setContext(RedisService.name);
+		this.logger.setContext(RedisFactory.name);
```

Comment:
Ich denke hier sollte mit dem token von Redis.Module.registerFor gearbeitet werden. Damit wir wissen in welchen Kontext die Logs entstehen. 
Ich würde zudem auch die Schreibweise im Construtor anpassen. Das private readonly entfernen, damit es nicht so wirkt als würde hier was injected werden.
```
logger.setContext(RedisFactory.name+context);
const redisFactory = new RedisFactory(config, logger);

// -------------------------------
config: RedisConfig;

logger: Logger;

public constructor(
		config: RedisConfig,
		logger: Logger,
		context: string
	) {
	this.config = config;
	this.logger = logger;
}     

```

Reply:
Set context habe ich ausgelagert. In constructor habe ich gelassen, weil das ist das selbe.